cmd/istioctl/main.go

```diff
@@ -403,6 +408,66 @@ and destination policies.
 		},
 	}
 
+	configCmd = &cobra.Command{
+		Use:   "context-create --api-server <ip>:<port>",
+		Short: "Create base kubeconfig file for a server",
+		Example: `
+		# Create a config file for the api server.
+		istioctl context-create --api-server http://127.0.0.1:8080
+		`,
+		RunE: func(c *cobra.Command, args []string) error {
+			if istioAPIServer == "" {
+				c.Println(c.UsageString())
+				return fmt.Errorf("specify the the Istio api server IP")
+			}
+
+			u, err := url.ParseRequestURI(istioAPIServer)
```

Comment:
As far as I tried, `<ip>:<port>` format will be invalid with `ParseRequestURI` or `Parse` (see https://play.golang.org/p/BbVPMSpKkh). Are we okay with that?  Since it's set through `u.String()` below, I'm unsure why it needs to be parsed here.

Reply:
@jmuk are you suggesting we not parse the server url value?  Or are you saying it is letting invalid URL's through or throwing errors for valid URLs?

Reply:
What is the expected format for the `Server` a URI or an authority?

Reply:
I'm saying `127.0.0.1:8080` would fall into parse error by `url.ParseRequestURI`. It needs to be `http://127.0.0.1:8080`. It is not clear to me this is intentional.

If it is intentional, please change the `Use` line above. If it's not, maybe we should fix this line. For the fixing, I'm wondering what format is expected in the `Server` field.  Are we okay to directly passing a string `"127.0.0.1:8080"`?